public/js/p3/store/IDMappingAppMemoryStore.js

```diff
@@ -161,7 +161,7 @@ define([
         // console.log(this.state);
 
         if(fromIdGroup === 'PATRIC'){
-          if(toIdGroup === 'PATRIC'){
+          if(toIdGroup === 'PATRIC' || toId === 'protein_id'){
```

Comment:
as far as i remember, there is a map to control what should be considered which "toIdGroup". Adding or in here will make it difficult to trace later.

Reply:
https://github.com/PATRIC3/p3_web/blob/develop/public/js/p3/widget/app/IDMapper.js#L129